File: Baekjoon/solved_ac/class2/30802.cpp
// 30802: 웰컴 키트

#include <iostream>
using namespace std;

int main() {
  int N, T, P, sum=0, shirts[6];
  cin >> N;

  for (int i=0; i<6; i++) cin >> shirts[i];
  cin >> T >> P;
  
  for (int i=0; i<6; i++) {
    if (shirts[i] % T == 0) sum += shirts[i]/T;
    else sum += shirts[i]/T + 1;
  }

  cout << sum << endl;
  cout << N/P << " " << N%P;

  return 0;
}

File: Baekjoon/solved_ac/class2/30802.py
# 30802: 웰컴 키트

N = int(input())
shirts = list(map(int, input().split()))
T, P = list(map(int, input().split()))

sum = 0
for i in range(6):
  if (shirts[i] % T == 0): sum += shirts[i]//T
  else: sum += shirts[i]//T + 1

print(sum)
print(N//P, N%P)
